src/PresentationalComponents/Labels/RuleLabels.js

```diff
@@ -1,52 +1,45 @@
 import './_RuleLabels.scss';
 
-import {
-  Tooltip,
-  TooltipPosition,
-} from '@patternfly/react-core/dist/esm/components/Tooltip/Tooltip';
+import { Tooltip, TooltipPosition, Label } from '@patternfly/react-core';
 
-import { Label } from '@patternfly/react-core/dist/esm/components/Label/Label';
 import PropTypes from 'prop-types';
 import React from 'react';
 import messages from '../../Messages';
-import { useIntl } from 'react-intl';
 
 const RuleLabels = ({ rule, isCompact, noMargin }) => {
-  const intl = useIntl();
-
   return (
     <React.Fragment>
       {rule?.tags?.search('incident') !== -1 && (
         <Tooltip
-          content={intl.formatMessage(messages.incidentTooltip)}
+          content={messages.incidentTooltip.defaultMessage}
```

Comment:
Yep, I see that intl itself is not being used which aligns with our aim. Especially for new components. However, we are still using messages object default messages in tests and other places. If you can find a better solution to share texts to avoid code duplication let's remove them. Otherwise, you can merge this and make improvements later.